frontend/appflowy_flutter/lib/plugins/shared/share/share_menu.dart

```diff
@@ -55,26 +56,40 @@ class _ShareMenuState extends State<ShareMenu>
       return const SizedBox.shrink();
     }
 
-    return Column(
-      mainAxisSize: MainAxisSize.min,
-      children: [
-        const VSpace(10),
-        Container(
-          alignment: Alignment.centerLeft,
-          height: 30,
-          child: _buildTabBar(context),
-        ),
-        Divider(
-          color: Theme.of(context).dividerColor,
-          height: 1,
-          thickness: 1,
-        ),
-        Padding(
-          padding: const EdgeInsets.symmetric(horizontal: 14.0),
-          child: _buildTab(context),
+    return FocusableActionDetector(
+      shortcuts: {
+        customCopyToClipboardCommand.command: const ActivateIntent(),
+      },
+      actions: {
+        ActivateIntent: CallbackAction<Intent>(
+          onInvoke: (intent) {
```

Comment:
The `onInvoke` function and the `handler` are both null, with only 'placeholder' comments.